ui/src/pages/Alert/index.tsx

```diff
@@ -35,8 +35,18 @@ const items: TabsProps['items'] = [
     label: '告警推送',
   },
 ];
+
+const getInitialActiveKey = () => {
+  const tailKey = location.hash.split('/')[location.hash.split('/').length - 1];
+  if (
+    ['event', 'shield', 'rules', 'channel', 'subscriptions'].includes(tailKey)
```

Comment:
Declare the array as a constant. If it does not change with the context, declare it outside the function.